.coveragerc

```diff
@@ -0,0 +1,18 @@
+[report]
+exclude_lines =
+	pragma: no cover
+	pragma: no unittest
```

Comment:
This will be excluded for all coverage measurements, not just unit tests.

Reply:
Yes, I believe it was just for a test build. Before we merge any of this the `pragma: no unittest` line needs to be removed.